### ISSUE TITLE
suggestion:Use #%% to execute a piece of code like a spyder when press alt+enter

### ISSUE THREAD

**Issue by schrodog** [pavanagrawal123/VSNotebooks#18]
Same as issue 163 in  VSNotebook repo, request for jupyter plugin like experience. Very important feature

**Comment by pavanagrawal123** [comment]
Yep, for sure, I definitely really want this as well. I will try to implement this next week!

**Comment by pavanagrawal123** [comment]
This is semi-done in #19!

**Comment by schrodog** [comment]
Still having issue with "VS Notebooks cannot find Jupyter in your custom environment ..."

**Comment by pavanagrawal123** [comment]
No, Neuron is supposed to be uninstalled when using VSNotebooks.

Can you share some screenshots on where the extension is starting the kernel and where your Jupyter executable is?

Thanks

**Comment by pavanagrawal123** [comment]
Ahh I see what the issue is, I will be pushing some changes to fix this in the beta branch!

**Comment by pavanagrawal123** [comment]
I published a beta version that fixes this issue! Try it out and let me know if it works for you!

**Comment by schrodog** [comment]
Great, now I works! Now I can press 'run cell' to get output.
Another issue: what is the shortcut key for "run cell"?

**Comment by pavanagrawal123** [comment]
That's awesome. I haven't implemented the shortcuts yet, I am still figuring that part out :) Will implement soon
